app/views/user/show.scala.html

```diff
@@ -126,7 +126,7 @@ <h1 class="lichess_title"><span@if(isOnline(u.id)) { class="connected" title="On
     <a class="intertab">@splitNumber(nb + " Blockers")</a>
     }
     <span class="intertab">@splitNumber(u.toints + " " + trans.tournamentPoints())</span>
-    <a class="intertab" href="@routes.ForumPost.search("user:" + u.username, 1)">@splitNumber(info.nbPosts + " Forum posts")</a>
+    <a class="intertab" href="@routes.ForumPost.search("user:" + u.username, 1)">@splitNumber(info.nbPosts + " " + trans.forumPosts())</a>
```

Comment:
this should be parameterized as `trans.nbForumPosts(info.nbPosts)`

Look for examples in the messages file.